### PR TITLE
refactor(switch): implement component

### PR DESCRIPTION
**Pull request by aesteves60**

### CODE REVIEW

**Review by feoche — CHANGES_REQUESTED** [review]
## Documentation
- [x] Overview : Select first item by default

## Demo
- [x] Setting `isDisabled` resets the component

## Style
- [x] Focus' style should be the same as the hover style + its focus outline

**Review by astagnol — CHANGES_REQUESTED** [review]
- For global UAT, see François Eoche comment
- Fix e2e test

**Review by dpellier — CHANGES_REQUESTED** [review]

File: packages/ods/src/components/switch/src/components/ods-switch/ods-switch.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import { Component, Element, Event, type EventEmitter, type FunctionalComponent, Host, Method, Prop, Watch, h } from '@stencil/core';
+import { type OdsRadioValueChangeEventDetail } from '../../../../radio/src';
+import { ODS_SWITCH_SIZE } from '../../constant/switch-size';
+import { clearOnItems, propagateIsDisabled, propagateSize, resetOnItems } from '../../controller/ods-switch';
+
+@Component({
+  shadow: true,
```
[the remark this answers is not in the document]
Comment (aesteves60) [reply packages/ods/src/components/switch/src/components/ods-switch/ods-switch.tsx:7]: With the radio, it's work with the form so I don't think it's relevent

**Review by astagnol — CHANGES_REQUESTED** [review]
- fix linters
- When an item is checked and the Switch is disabled, we should keep checked style with an opacity

File: packages/examples/react-webpack/package.json [hunk 2]
```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@ovhcloud/ods-components": "17.1.0",
+    "formik": "^2.4.6",
```
Comment (dpellier) [inline packages/examples/react-webpack/package.json:13]: no `^`

**Review by dpellier — CHANGES_REQUESTED** [review]

**Review by dpellier — CHANGES_REQUESTED** [review]
Still need to handle onKeyDown for Space to prevent scroll down on Firefox

**Review by dpellier — APPROVED** [review]

**Review by feoche — APPROVED** [review]

**Review by astagnol — APPROVED** [review]

**Pull request merged**